engine/fixtureParser.js

```
@@ -19,6 +19,11 @@ export default class FixtureParser {
 
   readFile(src) {
     const meta = matter.read(src);
+
+    if (Object.keys(meta.data).length === 0) {
+        throw new Error(`Error while parsing '${src}'.`);
```

I see, so it is actually `matter` that is failing silently. Fair enough.